charts/hono/config/router/qdrouterd.json

```diff
@@ -148,11 +146,12 @@
           "users": "Eclipse IoT;Hono",
           "remoteHosts": "*",
           "maxSessions": 4,
+          "maxSessionWindow": 8192000,
           "maxMessageSize": 131072,
           "allowUserIdProxy": true,
           "allowAnonymousSender": true,
-          "sources": "control/*, command/*",
-          "targets": "telemetry/*, event/*, control/*, command_response/*"
```

Comment:
The `control/*` addresses are still needed as of now (Hono 1.1).
(In Hono 1.2, that pattern will be renamed as part of the changes for https://github.com/eclipse/hono/issues/1272.)

Reply:
This is still used for forwarding commands that need to be mapped to a gateway ID, right?

Reply:
yes